### DISCUSSION TITLE
can we access the http server in wasm from other client by using http://127.0.0.1:8080/api/v1?

### DISCUSSION THREAD

**Discussion by aaronchen2k** [nlepage/go-wasm-http-server#14] [Q&A] [answered]
can we access the http server in wasm from other client by using url like http://127.0.0.1:8080/api/v1?

**Comment by aaronchen2k** [comment]
up

**Comment by nlepage** [comment, marked as answer]
Hi,

Some points :
 - Avoid "upping" issues please
 - Consider using the Discussions tab for questions
 - Avoid picking random labels such as "Enhancement" when you are just asking a question

This library emulates an HTTP server inside of a service worker, there is no actual HTTP server.
A service worker cannot be accessed from another client, neither can the emulated HTTP server.
